apps/web/modules/auth/lib/user.ts

```diff
@@ -43,6 +43,40 @@ export const updateUser = async (id: string, data: TUserUpdateInput) => {
   }
 };
 
+export const updateUserByEmail = async (email: string, data: TUserUpdateInput) => {
```

Comment:
it seems like this function is only used to update the `lastLoginAt` field. Let's make it more specific to that, e.g. `updateUserLastLoginAt(lastLoginAt: string):Promise<void>`. Also if it doesn't require returning fields, let's also remove them.

Otherwise we will run into issues with duplicated code and also need to test more code that we are not using.